client/dashboard/components/flash-message/index.tsx

```diff
@@ -36,7 +47,7 @@ export default function FlashMessage( {
 					break;
 			}
 
-			params.delete( key );
+			params.delete( overrideDefaultParam );
```

Comment:
Should this be `id` instead of `overrideDefaultParam`?

```suggestion
			params.delete( id );
```

Reply:
Nice catch. We had a few back and forth, and I missed this one...

Reply:
Done in f8a8008ce725d11eedaa2b79c00da3ed74316396